extensions/Vendor/EXT_animation_map/README.md

```diff
@@ -0,0 +1,94 @@
+# EXT_animation_map 
+
+## Contributors
+
+* Michael Feldstein, Facebook, [@msfeldstein](https://twitter.com/msfeldstein)
+* Ocean Quigley, Facebook, [@oceanquigley](https://twitter.com/oceanquigley)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF draft 2.0 spec.
+
+## Overview
+
+This extensions maps optional semantic meaning to animations so that client applications can attach interactive and reactive behaviors to the 3d models and scenes. This can be useful to let a client know to play a certain animation when the model first enters a scene, or when a model is about to leave.
+
+Facebook uses it to play animations when a model enters/leaves the feed, or scrubs an animation as the user scrolls the model through the screen.
+It is also used to respond to user events, such as tapping, or horizontal panning
+
+It maps a semantic, which can be any string a client wishes to bind, to an array of animation indices, pointing to a list of gltf Animation.
+
+##### Example Semantics
+
+* **ENTER**: Trigger a single shot animation when the object first enters the scene or viewport
+* **LEAVE**: Trigger a single shot animation as the object is leaving the scene or viewport
+* **ALWAYS**: Constantly loop an animation
+
+The following semantics require a list of glTF Node objects to specify interaction targets.  The event is triggered by acting on any of the nodes listed.  If the nodes are omitted, it can be assumed that the entire glTF Scene is the target.
```

Comment:
I find this confusing — the section is labeled "example semantics", which leads me to believe it is not normative, but the language here claims these semantics "require" a list of nodes and presumably must be used in a particular way. 

I'm guessing that none of this section's content it is meant to be normative, but either way I think this should be spelled out somewhat.

Reply:
Ah i see the confusion here.  The semantics above can operate without any target nodes since they act on the entire object, whereas the ones below need some more info (which nodes to act on).  Perhaps better wording would be "Semantics can have a list of nodes indicating which nodes the semantic should act upon if needed".  Lemme try to rewrite this